src/Sulu/Component/Content/Metadata/Loader/XmlLoader.php

```diff
@@ -76,6 +76,9 @@ private function createSection($propertyName, $data)
         if (isset($data['meta']['title'])) {
             $section->title = $data['meta']['title'];
         }
+        if (isset($data['meta']['info_text'])) {
+            $section->description = $data['meta']['info_text'];
+        }
```

Comment:
Could you expand the test for that? There's already a TODO in the `XmlLoaderTest` for that.